### DISCUSSION TITLE
Require packages in this list to have documentation

### DISCUSSION THREAD

**Discussion by JimmyCushnie** [starikcetin/awesome-upm#41] [Meta]
While working on #40, I had to go through every package in this list individually. A shocking number of them have nearly empty readmes; for many of the packages, I'm still not 100% sure what they do.

I propose the following as the minimum requirements for a package to be in this list:

1. The readme contains or links to a clear and obvious description of what the package does, and why it is useful
1. The readme contains or links to information about how to use the package -- i.e. code examples or screenshots of the Unity editor

This would probably cut the list in half, but IMO quality is more important than quality. This is supposed to be "a curated list" after all.

---

If we decide to go through with these or other minimum requirements:

1. I should not be the one to decide which packages are removed, due to conflict of interest. I am the author of many packages in the current list, and I am a contributer to many more.
1. For all packages that are removed, an issue should be opened in that repository to request that the issues be fixed.

**Comment by starikcetin** [comment]
I am kind of inbetween about this. Let's see what other people think.

PS: I am guilty of this myself as well, many of the packages of mine have little to no explanation. I'll go around fixing them all soon.
